erpnext_com/www/docs/user/manual/en/setting-up/print/raw-printing.md

```diff
@@ -0,0 +1,93 @@
+# Raw Printing
+
+> Introduced in Version 12
+
+### 1. Introduction to Raw Printing
+
+Sending a string of commands to a printer directly in its native language is called Raw Printing. Many thermal printers need these raw commands sent to them in order to perform functions like barcode printing, receipt printing, label printing, etc. Raw Printing bypasses the printer's drivers in most cases, making it very fast and reliable. Raw Printing is also capable of doing some advanced features such as cutting receipt paper, kicking out cash drawers, etc.
+
+### 2. Setting up Raw Printing in ERPNext
```

Comment:
so now line 9 will be "1.", make subsequent changes

Reply:
done...